src/components/HomePageContent/HowItWorksSection/TransferableRecordPersonas.tsx

```diff
@@ -4,80 +4,80 @@ export const TransferableRecordPersonas: Persona[] = [
   {
     image: "/static/images/home//howItWorks/persona/persona1.png",
     jobTitle: "The Exporter",
-    description: "This is Kevin, he is an exporter with customers in many different countries",
+    description:
+      "This is Kevin, he is an exporter with customers in many different countries. See how he cut down on time and cost when dealing with Blank-Endorsed BL.",
     learnMore: {
       title: "Electronic Bill of Lading",
       thenSteps: [
         {
-          stepNumber: "1",
+          stepTitle: "Step 1",
           icon: "/static/images/home/howItWorks/icons/paperReceive.svg",
           description: "Exporter received BL from carrier (via courier)",
         },
         {
-          stepNumber: "2",
+          stepTitle: "Step 2",
           icon: "/static/images/home/howItWorks/icons/paperCheck.svg",
           description: "Exporter checks the details of BL including entire endorsement chain",
         },
         {
-          stepNumber: "3",
+          stepTitle: "Step 3",
           icon: "/static/images/home/howItWorks/icons/paperEndorse.svg",
           description: "Exporter endorses BL",
         },
         {
-          stepNumber: "4",
+          stepTitle: "Step 4",
           icon: "/static/images/home/howItWorks/icons/paperDispatch.svg",
           description: "Exporter dispatches the BL to Negotiating Bank (via courier)",
         },
       ],
       nowSteps: [
         {
-          stepNumber: "1",
+          stepTitle: "Step 1",
           icon: "/static/images/home/howItWorks/icons/eReceive.svg",
           description: "Exporter receives eBL from carrier (via email)",
         },
         {
-          stepNumber: "2",
+          stepTitle: "Step 2",
           icon: "/static/images/home/howItWorks/icons/eCheck.svg",
           description: "Exporter checks for authenticity and provenance including endorsement entry",
         },
         {
-          stepNumber: "3",
+          stepTitle: "Step 3",
           icon: "/static/images/home/howItWorks/icons/eEndorse.svg",
           description: "Exporter performs endorsement to Negotiating Bank",
         },
       ],
-      endMessage: "These TradeTrust benefits are just the tip of the iceberg. Get in touch to find out more!",
+      endMessage: "TradeTrust is more than a tool to hasten your exporting processes. Get in touch to find out more!",
     },
   },
   {
     image: "/static/images/home/howItWorks/persona/persona2.png",
     jobTitle: "The Carrier",
     description:
-      "This is Lizzie, she works for a Carrier, and her Company is using TradeTrust for their e-Bill-of-Lading Solution",
+      "This is Lizzie and she works for a Carrier. She uses an electronic Bill-of-Lading (eBL) solution enabled by TradeTrust to issue and surrender eBLs to her clients.",
     learnMore: {
       title: "Electronic Bill of Lading",
```

Comment:
update to: "Issuance and Surrender of Electronic Bill of Lading"

Reply:
done.